server/src/main/java/com/paypal/selion/utils/SauceLabsRestApiFactory.java

```diff
@@ -0,0 +1,144 @@
+/*-------------------------------------------------------------------------------------------------------------------*\
+|  Copyright (C) 2016 PayPal                                                                                          |
+|                                                                                                                     |
+|  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance     |
+|  with the License.                                                                                                  |
+|                                                                                                                     |
+|  You may obtain a copy of the License at                                                                            |
+|                                                                                                                     |
+|       http://www.apache.org/licenses/LICENSE-2.0                                                                    |
+|                                                                                                                     |
+|  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed   |
+|  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for  |
+|  the specific language governing permissions and limitations under the License.                                     |
+\*-------------------------------------------------------------------------------------------------------------------*/
+
+package com.paypal.selion.utils;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonSyntaxException;
+import com.paypal.selion.logging.SeLionGridLogger;
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.HttpStatus;
+import org.openqa.grid.common.exception.GridConfigurationException;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.logging.Level;
+
+/**
+ * A simple helper class for querying some of the sauce labs account details via their rest api.
+ */
+public class SauceLabsRestApiFactory {
```

Comment:
yea.. I struggled with this also.. "helper" didn't fit either.. I'll just drop "factory" and call it good enough unless you have a better suggestion.